examples/mplot3d/view_planes_3d.py

```diff
@@ -0,0 +1,60 @@
+"""
+======================
+Primary 3D view planes
+======================
+
+This example generates an "unfolded" 3D plot that shows each of the primary 3D
+view planes. The elevation, azimuth, and roll angles required for each view are
+labeled. You could print out this image and fold it into a box where each plane
+forms a side of the box.
+"""
+
+import matplotlib.pyplot as plt
+
+
+def annotate_axes(ax, text, fontsize=18):
+    ax.text(x=0.5, y=0.5, z=0.5, s=text,
+            va="center", ha="center", fontsize=fontsize, color="black")
+
+# (view, (elev, azim, roll))
+views = [('XY',   (90, -90, 0)),
+         ('XZ',    (0, -90, 0)),
+         ('YZ',    (0,   0, 0)),
+         ('-XY', (-90,  90, 0)),
+         ('-XZ',   (0,  90, 0)),
+         ('-YZ',   (0, 180, 0))]
+
+layout = [['XY',  '.',   'L',   '.'],
+          ['XZ', 'YZ', '-XZ', '-YZ'],
+          ['.',   '.', '-XY',   '.']]
+fig, axd = plt.subplot_mosaic(layout, subplot_kw={'projection': '3d'},
+                              figsize=(12, 8.5))
+for i in range(len(axd) - 1):
+    plane = views[i][0]
```

Comment:
We don't need indices:
```suggestion
for plane, angles in views:
```